### ISSUE TITLE
Donation total not respect to the number of decimal

### ISSUE THREAD

**Issue by jaydeeprami** [WordImpress/Give#2461]
## Issue Overview

- Donation total does not respect to the number of decimal while switching on donation amount value.
- Same with Custom amount
- Issue exist on latest Give 1.8.17

![nimbus-record-video-2017-12-06-11-07-12](https://user-images.githubusercontent.com/15060983/33646822-a5fac464-da77-11e7-8dcc-fcd622ffee60.gif)

## Expected Behavior

- The number of decimal should remain same when switching to the donation amount or gateway.

## Current Behavior

- Currently, It does not remain same and change from  $10.00 to $10 while switching to the amount.
- Also with the Custom amount, it does not work.

## Steps to Reproduce (for bugs)

1. Create a donation form with Multi-level donation with various amount
2. Switching among the amount 
3. As you can see that decimal value does not remain same when switching

## Todos
- [ ] Tests

## WordPress Environment
<details>
```
### WordPress Environment ###

Home URL: http://givewp.dev
Site URL: http://givewp.dev
WP Version: 4.9.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
Show on Front: posts
Table Prefix Length: 3
Table Prefix Status: Acceptable
Admin AJAX: Inaccessible
Registered Post Statuses: publish, future, draft, pending, private, trash, auto-draft, inherit, refunded, failed, revoked, cancelled, abandoned, processing, preapproval

### Server Environment ###

Hosting Provider: DBH: localhost, SRV: givewp.dev
TLS Connection: Connection uses TLS 1.2
TLS Connection: Probably Okay
Server Info: nginx/1.10.1
PHP Version: ❌ 5.3.29 - We recommend a minimum PHP version of 5.6. See: PHP Requirements in Give
PHP Post Max Size: 1,000 MB
PHP Time Limit: 900
PHP Max Input Vars: 3000
PHP Max Upload Size: 1,000 MB
cURL Version: ❌ 7.38.0, OpenSSL/1.0.2h - We recommend a minimum cURL version of 7.40.
SUHOSIN Installed: –
MySQL Version: ❌ 5.5.55 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
gzip: ✔
GD Graphics Library: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Give Configuration ###

Give Version: 1.8.17
Upgraded From: 1.8.17
Test Mode: Enabled
Currency Code: INR
Currency Position: Before
Decimal Separator: .
Thousands Separator: ,
Success Page: http://givewp.dev/donation-confirmation/
Failure Page: http://givewp.dev/donation-failed/
Donation History Page: http://givewp.dev/donation-history/
Give Forms Slug: /donations/
Enabled Payment Gateways: Test Donation, Offline Donation, PayPal Standard
Default Payment Gateway: Test Donation
PayPal IPN Verification: Enabled
PayPal IPN Notifications: N/A
Admin Email Notifications: Disabled
Donor Email Access: Enabled

### Session Configuration ###

Give Use Sessions: Enabled
Session: Disabled

### Active Give Add-ons ###


### Other Active Plugins ###


### Inactive Plugins ###

Give - Currency Switcher: by WordImpress – 1.0.0
Give - Fee Recovery: by WordImpress – 1.3.2
Give - Form Field Manager: by WordImpress – 1.2.3
Give - Gift Aid: by WordImpress – 1.0.0
Give - PDF Receipts: by WordImpress – 2.1
Give - Recurring Donations: by WordImpress – 1.4
Give - Setting API Fields Example: by WordImpress – 1.0
Give - Stripe Gateway: by WordImpress – 1.5
Give - Tributes: by WordImpress – 1.2

### Active MU Plugins ###

Local by Flywheel Relative URL (for Live Links): by Flywheel – 1.0

### Theme ###

Name: Twenty Seventeen
Version: 1.4
Author URL: https://wordpress.org/
Child Theme: No – If you're modifying Give on a parent theme you didn't build personally, then we recommend using a child theme. See: How to Create a Child Theme

```
</details>

**Comment by raftaar1191** [comment]
I think it's okay as .00 is not so important

**Comment by DevinWalker** [comment]
@raftaar1191 it is important but @jaydeeprami I am unable to reproduce. Can you try on a fresh install and create a video? I'm suspect that this is an conflict with another add-on or plugin and not a core issue.

**Comment by jaydeeprami** [comment]
Sure @DevinWalker , I will create a video of fresh install.

**Comment by jaydeeprami** [comment]
@DevinWalker, I have just checked with the Fresh install and cannot reproduce this issue. Also, In my Previous install, there is a no any Add-on activated.

**Comment by jaydeeprami** [comment]
@DevinWalker, I can able to reproduce this issue with Indian Rupees.

**Comment by raftaar1191** [comment]
@jaydeeprami Can you share the GIve Setting from the old sites by export it from the Tools > Export > Export Give Settings

**Comment by DevinWalker** [comment]
@jaydeeprami this appears resolved now after merging #2462 - can you pull down the latest and confirm?

**Comment by jaydeeprami** [comment]
@DevinWalker, Yes, it's resolved.
